src/Statistics.jl

```diff
@@ -630,7 +653,13 @@ function cov2cor!(C::AbstractMatrix, xsd::AbstractArray, ysd::AbstractArray)
 end
 
 # corzm (non-exported, with centered data)
-
+function corzm(itr::Any) 
```

Comment:
Can you put this code in an internal method which will be called by all functions that need it? It's repeated three times.

Also:
```suggestion

function corzm(itr::Any) 
```